php/libraries/NDB_Config.class.inc

```diff
@@ -570,9 +572,7 @@ class NDB_Config
             $nextLevel = NDB_Config::getMenuTabs($thisRow['ID']);
 
             if (!empty($nextLevel)) {
-                if (NDB_Config::checkMenuPermission($nextLevel[0]['ID'])) {
```

Comment:
@jstirling91 any variable that has been return by the `getMenuTabs($thisRow['ID']);` call on line 570 has already underwent this check on line 559 otherwise it would not get returned into `$nextLevel`

let me know if you disagree